src/authV2/azext_authV2/_help.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['webapp auth'] = """
+type: group
+short-summary: Manage webapp authentication and authorization in the v2 format.
+"""
+
+helps['webapp auth show'] = """
+type: command
+short-summary: Show the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Show the authentication settings for the webapp. (autogenerated)
+    text: az webapp auth show --name MyWebApp --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['webapp auth update'] = """
+type: command
+short-summary: Update the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Update the client ID of the AAD provider already configured
+    text: >
+        az webapp auth update  -g myResourceGroup -n myUniqueApp --set identityProviders.azureActiveDirectory.registration.clientId=ecbacb08-df8b-450d-82b3-3fced03f2b27
+  - name: Enable authentication on the app
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --enabled true
+  - name: Pin the runtime version of the app to 1.4.7
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --runtime-version 1.4.7
+  - name: Configure the app with file based authentication by setting the config file path
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --config-file-path auth.json
+  - name: Configure the app to redirect to AzureActiveDirectory's login page for unauthenticated requests
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --unauthenticated-client-action RedirectToLoginPage --redirect-provider AzureActiveDirectory
```

Comment:
Can you remind me what the default setup is? If `RedirectToLoginPage` is already the default, then I recommend the example be something different. However, if it's _not_ the default, then this is fine. We might even consider bumping this up higher in the list since it could be a more common operation compared to the others.

Reply:
yes default is RedirectToLoginPage, so I'll change the example in that case.